orchagent/portsorch.cpp

```diff
@@ -4761,7 +4913,11 @@ void PortsOrch::doTask(Consumer &consumer)
 
     string table_name = consumer.getTableName();
 
-    if (table_name == APP_PORT_TABLE_NAME)
+    if (table_name == STATE_TRANSCEIVER_INFO_TABLE_NAME && m_cmisModuleAsicSyncSupported)
```

Comment:
Do we need check `m_cmisModuleAsicSyncSupported` here? We will only subscribe transceiver table if m_cmisModuleAsicSyncSupported is true.